### ISSUE TITLE
Rework the Repair Sign Mechanics

### ISSUE THREAD

**Issue by KingSasukePT** [APDevTeam/Movecraft-Repair#10]
As today, the current way the Repair Sign allows players to both Save, Repair and Construct (with saved "files") any building or craft as long as ALL the materials are available in a chest inside the Shipyard. It's in my fairest belief that the player should be able to start the construction of ANY building or craft with the available material whether or not ALL that are needed are available. One but not only of the major pros of this idea is the fact that it can override some itens that were in the saved "file" but that the player does not have or cannot get anymore.

**Comment by TylerS1066** [comment]
This is an interesting concept.  I've added it to the list of features we plan on someday implementing but sadly it is a low priority so unlikely to be implemented any time soon.  This also depends on https://github.com/APDevTeam/Movecraft-Repair/issues/4 to redo the entire user experience.

**Comment by KingSasukePT** [comment]
> 
> 
> This is an interesting concept. I've added it to the list of features we plan on someday implementing but sadly it is a low priority so unlikely to be implemented any time soon. This also depends on #4 to redo the entire user experience.

Yeah but as long as the concept are already proposed, it might just be a matter of time. :)

**Comment by AdmiralHowe** [comment]
Perhaps consider using fireworks as an item to differentiate between saving a state and building one, to make it easier to avoid accidentally overwriting your own states.

Also, another potential suggestion would be to make the sign place a highlight or outline of some kind when you first click the sign to start building, so you can tell where the state will be constructed.

**Comment by TylerS1066** [comment]
Note: This, #5 and #4 are all tied closely together.
